src/settings/Settings.js

```diff
@@ -120,6 +120,12 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "feature_event_indexing": {
+        isFeature: true,
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
```

Comment:
This should be `LEVELS_FEATURE`, unless there's something special happening for this feature...?

Reply:
Fixed.